package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "monday-ui-react-core",
-  "version": "0.3.49",
+  "version": "0.3.51",
```

Comment:
still need to figure out:
- shimmer animation
- a11y - how to handle skeletons?
- all grey color key (replace for black mode)